doc/tools/docgen_cmd.py

```diff
@@ -60,18 +62,16 @@ def abort(error):
     exit()
 
 
-def get_rst_string(module_name, help_string):
-    """
-    Generate rst text for module
-    """
-    dashes = "========================\n"
+def get_help_string(class_obj):
+    # return inspect.getdoc(class_obj.run)
+    try:
+        ia_module = importlib.import_module("dipy.workflows.base")
```

Comment:
why you cannot use `ia_module = dipy.workflows.base` ?

Reply:
Hi @Borda, Thank you for your review.

I do not understand your point here and how it will work without importing the module. Can you give me more detail?

Reply:
the question is why are you using `ia_module = importlib.import_module("dipy.workflows.base")` instead of `ia_module = dipy.workflows.base` ?

Reply:
I need to import the module. if I do `ia_module = dipy.workflows.base` the module is not imported right or I miss something?

Reply:
what do you mean by right imported? `import dipy.workflows.base` works firn for me...